.gitlab-ci.yml

```diff
@@ -12,3 +11,32 @@ build and push emulator image:
   script:
     - docker-compose -f ./docker-compose.yml build
     - docker-compose -f ./docker-compose.yml push
+
+build test:
+  stage: test
+  image: golang
+  variables:
+    TEST: "build-local"
+  script:
+    - go version
+    - go build .
+    - go test -v ./...
+
+go lint:
+  stage: test
+  image: golang
+  variables:
+    TEST: "golangci"
```

Comment:
What does this do?

Reply:
This will run linter for golang and will check the files.

Reply:
I mean not the job but just the `TEST` variable. It's not used explicitly in the script.

Is it picked up implicitly by `golangci-lint` or the install scripts?

Reply:
That might be unnecessary in gitlab ci, pushed a commit without the variables to test if the gitlab ci finishes correctly. Thanks for pointing it out.

Reply:
Thx. Resolving.